package.json

```diff
@@ -11,7 +11,9 @@
     "eject": "react-scripts eject",
     "fmt": "prettier --check '**/*.ts'",
     "fmt:write": "prettier --write '**/*.ts'",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "generate-types": "typechain --target=ethers-v5 --out-dir src/contracts './node_modules/@openzeppelin/contracts/build/contracts/IERC20.json'",
```

Comment:
Maybe this should be used in yarn build if it's necessary.

Reply:
Thats why I put it in postinstall. Thats where the doc / example of typechain suggested to put it.
Postinstall is a [lifecycle script](https://yarnpkg.com/advanced/lifecycle-scripts) from yarn:
> postinstall is called after a package got successfully installed on the disk. It is guaranteed.

So if you just do yarn install / yarn it will automatically execute the generate-types script too